cmd/main.go

```diff
@@ -0,0 +1,122 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/jmoiron/sqlx"
+	"github.com/mainflux/et/internal"
+	authClient "github.com/mainflux/et/internal/clients/grpc/auth"
+	jaegerClient "github.com/mainflux/et/internal/clients/jaeger"
+	"github.com/mainflux/et/internal/env"
+	"github.com/mainflux/et/internal/homing"
+	"github.com/mainflux/et/internal/homing/api"
+	"github.com/mainflux/et/internal/homing/repository/sheets"
+	"github.com/mainflux/et/internal/homing/repository/timescale"
+	"github.com/mainflux/et/internal/server"
+	httpserver "github.com/mainflux/et/internal/server/http"
+	"github.com/mainflux/mainflux"
+	mflog "github.com/mainflux/mainflux/logger"
+	"golang.org/x/sync/errgroup"
+)
+
+const (
+	svcName        = "homing"
+	envPrefix      = "MF_HOMING_"
+	envPrefixHttp  = "MF_HOMING_"
+	defSvcHttpPort = "8855"
+)
+
+type config struct {
+	LogLevel       string `env:"MF_USERS_LOG_LEVEL"  envDefault:"info"`
+	JaegerURL      string `env:"MF_JAEGER_URL"       envDefault:"localhost:6831"`
+	GCPCredFile    string `env:"MF_GCP_CRED"`
+	SpreadsheetId  string `env:"MF_SPREADSHEET_ID"`
+	SheetId        int    `env:"MF_SHEET_ID"         envDefault:"0"`
+	IPDatabaseFile string `env:"MF_IP_DB"`
+}
+
+func main() {
+	ctx, cancel := context.WithCancel(context.Background())
+	g, ctx := errgroup.WithContext(ctx)
+
+	cfg := config{}
+	if err := env.Parse(&cfg); err != nil {
+		log.Fatalf("failed to load %s configuration : %s", svcName, err)
+	}
+
+	timescaleConf := timescale.Config{}
+	if err := env.Parse(&timescaleConf); err != nil {
+		log.Fatalf("failed to load %s timescale configuration : %s", svcName, err)
+	}
+
+	timescaleDB, err := timescale.Connect(timescaleConf)
+	if err != nil {
+		log.Fatalf("failed to connect to timescale db : %s", err)
+	}
+
+	logger, err := mflog.New(os.Stdout, cfg.LogLevel)
```

Comment:
You can move this after env parsing then use `logger` thererafter rather than `log.`

Reply:
Change the version of the `logger` you are importing. The latest version has `logger.Fatal`